tests/Aura/Uri/CheckPublicSuffixTest.php

```diff
@@ -64,9 +67,9 @@ public function testPublicSuffixSpec()
         $this->checkPublicSuffix('test.ac', 'test.ac');
         // TLD with only 1 (wildcard) rule.
         $this->checkPublicSuffix('cy', null);
-        $this->checkPublicSuffix('c.cy', null);
-        $this->checkPublicSuffix('b.c.cy', 'b.c.cy');
-        $this->checkPublicSuffix('a.b.c.cy', 'b.c.cy');
+        $this->checkPublicSuffix('c.cy', 'c.cy');
```

Comment:
Does this test case changed ?

Reply:
Because the source data seems change. Related https://github.com/auraphp/Aura.Uri/pull/33/commits/1d3df22ce573f212c004f58f30509c1f1496abc0